src/Channel.h

```diff
@@ -71,7 +71,23 @@ class Channel : public QObject {
 
 #ifdef MUMBLE
 	unsigned int uiPermissions;
-	bool bFiltered;
+
+	/// Visibility modifiers used when applying the channel filter
+	/// The channel the user is in will be always visible
+	/// This enum is used in the client DB, only append new entries
+	enum FilterMode {
+		/// Channel is filtered, if it is empty and 'Automatically hide empty channels' is active
```

Comment:
I wouldn't go into much detail on the default filter behavior - this is only prone to becoming outdated eventually
```suggestion
		/// The default filtering behavior
```